app/controllers/circles_controller.rb

```diff
@@ -23,6 +24,47 @@ def create
     end
   end
 
+  def edit
+    unless user_is_creator?
+      flash[:error] = ["Sorry, something went wrong. Please try again",
+                      "You didn't create this circle so you can't edit its details!"]
```

Comment:
Line is too long. [85/80]<br>Align the elements of an array literal if they span more than one line.